integration/release/release.go

```diff
@@ -37,9 +37,9 @@ func New(config Config) (*Release, error) {
 	return r, nil
 }
 
-func (r *Release) WaitForChartInfo(ctx context.Context, release, version string) error {
+func (r *Release) WaitForChartInfo(ctx context.Context, namespace, release, version string) error {
```

Comment:
All methods in Helm 3 are namespaced. So we need extra params. We get the namespace from the chart CR.